src/spclient.rs

```diff
@@ -353,18 +363,23 @@ impl SpClient {
         if let Some(deduce_from) = payer_vouts.choose(&mut rng) {
```

Comment:
Later we should make it so that it will only deduct fees from the change address (if it exists)

Reply:
Actually I like having the option to deduce from another outputs, even if the change (when it exists) is the obvious default.

Reply:
Made a new issue for this